src/dialog/Dialog/Dialog.tsx

```diff
@@ -52,7 +52,7 @@ import {
     ValidatorType,
 } from '../types';
 
-const bDialog = cn('df-dialog');
+const bDialog = dfCN('dialog');
 const bPage = cn('page-dialog');
```

Comment:
Ok. I fix it but it's changes className for this block and it can create some side effects in services that use this package. For example if they wrote styles or tests using this className.

Reply:
Probably we need to release it as a major version. There is no squashing configured for commits of PRs please leave only one commit with corresponding title/comment.
Is it the only place where `cn`-variable is used (except stories)? If so we can remove it and leave only dfCN, and do renaming `dfCN` => `cn`.

Reply:
But if you think if `dfCN` is better than just `cn` i think it is OK too, so then rework only commits please.